Launcher/Classes/RegexHandler.swift

```diff
@@ -9,21 +9,22 @@
 import Foundation
 
 class RegexHandler {
-    func matchesForRegexInText(regex: String!, text: String!) -> [String] {
+    func matchesForRegexInText(regex: String, text: String, global: Bool = false) -> [String] {
```

Comment:
What about `matches(for regex: String, in text: String, global: Bool = false) -> [String]`?
This will then be used as:

```swift
RegexHandler().matches(for: "\\u001B\\[([^m]*)m", in: try_mathc, global: true)
```

Also, what does the `global` parameter do exactly?